openedx/features/course_experience/views/course_updates.py

```diff
@@ -0,0 +1,64 @@
+"""
+Views that handle course updates.
+"""
+
+from django.contrib.auth.decorators import login_required
+from django.core.context_processors import csrf
+from django.core.urlresolvers import reverse
+from django.template.loader import render_to_string
+from django.utils.decorators import method_decorator
+from django.views.decorators.cache import cache_control
+
+from courseware.courses import get_course_info_section, get_course_with_access
+from lms.djangoapps.courseware.tabs import CoursewareTab
+from lms.djangoapps.courseware.views.views import CourseTabView
+from opaque_keys.edx.keys import CourseKey
+from openedx.core.djangoapps.plugin_api.views import EdxFragmentView
+from web_fragments.fragment import Fragment
+
+
+class CourseUpdatesView(CourseTabView):
+    """
+    The course updates page.
+    """
+    @method_decorator(login_required)
+    @method_decorator(cache_control(no_cache=True, no_store=True, must_revalidate=True))
+    def get(self, request, course_id, **kwargs):
+        """
+        Displays the home page for the specified course.
+        """
+        return super(CourseUpdatesView, self).get(request, course_id, 'courseware', **kwargs)
+
+    def render_to_fragment(self, request, course=None, tab=None, **kwargs):
+        course_id = unicode(course.id)
```

Comment:
Fragment views have to take strings for all their parameters because they are exposed as URLs. So to call the fragment in process, we have to pass the course_id. It is too late to change this now, but it does make it seem odd that this method takes a course and not a course_id to be consistent with fragment views.